ir/v1.def

```diff
@@ -230,7 +230,7 @@ class CalculatedField : IAnnotated {
         bool                    update;
         ID                      name;
         Expression              cond;
-        update_or_verify() {}  // FIXME -- needed by umpack_json(safe_vector) -- should not be
+        update_or_verify() { update = false; }  // FIXME -- needed by umpack_json(safe_vector) -- should not be
```

Comment:
Should this be in the function call or in the initializer of `update`?

Reply:
I would like to put it into the initializer, but for some reason that breaks the generated code -- the parametric constructor has just 3 arguments. I don't know why that is, it seems quite weird to me. Therefore I've added it to the constructor. I'm quite surprised nested classes can be generated by IRGen.

Reply:
`optional bool           update = false;`
maybe?

Reply:
That worked, thanks.